cypress/integration/app-init-spec.js

```diff
@@ -0,0 +1,13 @@
+describe('app Initialization', () => {
+  Cypress.on('uncaught:exception', (err, runnable) => false);
+
+  it('Loads movieMockData on page load', () => {
+    beforeEach(() => {
```

Comment:
I don't think `beforeEach` should be in `it` block, it should be placed in `describe` - `it` is test case and can't have nested test cases

Reply:
Updated and It is fast Now